Traceback (most recent call last):
  File "C:\Users\Berend\anaconda3\envs\jupyterbook\lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Users\Berend\anaconda3\envs\jupyterbook\lib\site-packages\nbclient\client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\Berend\anaconda3\envs\jupyterbook\lib\site-packages\jupyter_core\utils\__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
  File "C:\Users\Berend\anaconda3\envs\jupyterbook\lib\asyncio\base_events.py", line 647, in run_until_complete
    return future.result()
  File "C:\Users\Berend\anaconda3\envs\jupyterbook\lib\site-packages\nbclient\client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "C:\Users\Berend\anaconda3\envs\jupyterbook\lib\site-packages\nbclient\client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\Berend\anaconda3\envs\jupyterbook\lib\site-packages\nbclient\client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# Maak een lege dataframe met de gewenste kolommen
df = pd.DataFrame(columns=['Welke graph', 'Welke AI', 'Waarom was het nodig', 'Wat gevraagd'])

# Voeg 7 rijen toe aan de dataframe met de gewenste waarden
for i in range(1, 8):
    df = df.append({
        'Welke graph': i,
        'Welke AI': 'Chat-GPT',
        'Waarom was het nodig': 'Aanvullen van code' if i % 2 == 1 else 'Helpen met error',
        'Wat gevraagd': [
            'Hoe ziet een basis landkaart er uit en welke imports heb ik nodig?',
            'Waarom krijg ik deze error (paste code) >> (bleek een verkeerde kleur module te zijn)',
            'Hoe maak je een bubbelplot',
            'Waarom krijg ik hier een error (paste code)',
            'Hoe maak je een boxplot in jupyter notebook?',
            'how to select all rows that have a string that starts with 2019 in the dt column',
            'how to export a df as csv using pandas'
        ][i-1]
    }, ignore_index=True)

# Verberg de index van de dataframe
df.style.hide_index()

# Geef de tabel een titel
print('Appendix')
display(df)
------------------


[1;31m---------------------------------------------------------------------------[0m
[1;31mAttributeError[0m                            Traceback (most recent call last)
[1;32m~\AppData\Local\Temp\ipykernel_19864\1148475671.py[0m in [0;36m?[1;34m()[0m
[0;32m      5[0m [1;31m# Maak een lege dataframe met de gewenste kolommen[0m[1;33m[0m[1;33m[0m[0m
[1;32m----> 6[1;33m [0mdf[0m [1;33m=[0m [0mpd[0m[1;33m.[0m[0mDataFrame[0m[1;33m([0m[0mcolumns[0m[1;33m=[0m[1;33m[[0m[1;34m'Welke graph'[0m[1;33m,[0m [1;34m'Welke AI'[0m[1;33m,[0m [1;34m'Waarom was het nodig'[0m[1;33m,[0m [1;34m'Wat gevraagd'[0m[1;33m][0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m      7[0m [1;33m[0m[0m
[0;32m      8[0m [1;31m# Voeg 7 rijen toe aan de dataframe met de gewenste waarden[0m[1;33m[0m[1;33m[0m[0m
[0;32m      9[0m [1;32mfor[0m [0mi[0m [1;32min[0m [0mrange[0m[1;33m([0m[1;36m1[0m[1;33m,[0m [1;36m8[0m[1;33m)[0m[1;33m:[0m[1;33m[0m[1;33m[0m[0m

[1;32m~\anaconda3\envs\jupyterbook\lib\site-packages\pandas\core\generic.py[0m in [0;36m?[1;34m(self, name)[0m
[0;32m   5985[0m             [1;32mand[0m [0mname[0m [1;32mnot[0m [1;32min[0m [0mself[0m[1;33m.[0m[0m_accessors[0m[1;33m[0m[1;33m[0m[0m
[0;32m   5986[0m             [1;32mand[0m [0mself[0m[1;33m.[0m[0m_info_axis[0m[1;33m.[0m[0m_can_hold_identifiers_and_holds_name[0m[1;33m([0m[0mname[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0;32m   5987[0m         ):
[0;32m   5988[0m             [1;32mreturn[0m [0mself[0m[1;33m[[0m[0mname[0m[1;33m][0m[1;33m[0m[1;33m[0m[0m
[1;32m-> 5989[1;33m         [1;32mreturn[0m [0mobject[0m[1;33m.[0m[0m__getattribute__[0m[1;33m([0m[0mself[0m[1;33m,[0m [0mname[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0m
[1;31mAttributeError[0m: 'DataFrame' object has no attribute 'append'

